### PR TITLE
DOCS-969: Rename radius_cluster_segmenter

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/services/vision/segmentation.md [hunk 1]
```diff
@@ -98,14 +108,17 @@ Add the vision service object to the services array in your raw JSON configurati
...
+| `max_dist_from_plane_mm` | Optional | A float that determines how much area above and below an ideal ground plane should count as the plane for which points are removed. For fields with tall grass, this should be a high number. The default value is 100 mm. |
+| `ground_plane_normal_vec` | Optional | A `(x,y,z)` vector that represents the normal vector of the ground plane. Different cameras have different coordinate systems. For example, a lidar's ground plane will point in the `+z` direction `(0, 0, 1)`. On the other hand, the intel realsense `+z` direction points out of the camera lens, and its ground plane is in the negative y direction `(0, -1, 0)`. The default value is `(0, 0, 1)`. |
+| `ground_angle_tolerance_degs` | Optional | A float that determines how strictly the found ground plane should match the `ground_plane_normal_vec`. For example, even if the ideal ground plane is purely flat, a rover may encounter slopes and hills. The algorithm should find a ground plane even if the found plane is at a slant, up to a certain point. The default value is 30 degrees. |
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:121]:
```suggestion
| `ground_angle_tolerance_degs` | Optional | A float that determines how strictly the found ground plane should match the `ground_plane_normal_vec`. For example, even if the ideal ground plane is purely flat, a rover may encounter slopes and hills. The algorithm should find a ground plane even if the found plane is at a slant, up to a certain point. <br> Default: `30.00` </br> |
```

(Opt. if we are still doing this with andrew's table format updates you could do this the same way in the column above!)

**Review by sguequierre — CHANGES_REQUESTED** [review]
One typo otherwise lgtm

File: docs/services/vision/segmentation.md [hunk 2]
```diff
@@ -98,14 +108,17 @@ Add the vision service object to the services array in your raw JSON configurati
...
 | `mean_k_filtering` | _Optional_ | An integer parameter used in [a subroutine to eliminate the noise in the point clouds](https://pcl.readthedocs.io/projects/tutorials/en/latest/statistical_outlier.html). It should be set to be 5-10% of the number of min_points_in_segment. Start with 5% and go up if objects are still too noisy. If you don’t want to use the filtering, set the number to 0 or less. |
+| `max_dist_from_plane_mm` | Optional | A float that determines how much area above and below an ideal ground plane should count as the plane for which points are removed. For fields with tall grass, this should be a high number. The default value is 100 mm. |
+| `ground_plane_normal_vec` | Optional | A `(x,y,z)` vector that represents the normal vector of the ground plane. Different cameras have different coordinate systems. For example, a lidar's ground plane will point in the `+z` direction `(0, 0, 1)`. On the other hand, the intel realsense `+z` direction points out of the camera lens, and its ground plane is in the negative y direction `(0, -1, 0)`. The default value is `(0, 0, 1)`. |
```
[the remark this answers is not in the document]
Comment (bhaney) [reply docs/services/vision/segmentation.md:120]: Yes, these are the cartesian coordinates r3.vector. This is the arrow that points perpendicular to the ground plane.

**Review by bhaney — APPROVED** [review]

File: docs/services/vision/segmentation.md [hunk 3]
```diff
@@ -50,7 +50,10 @@ In your vision service's panel, fill in the **Attributes** field.
     "min_points_in_plane": <integer>,
     "min_points_in_segment": <integer>,
     "clustering_radius_mm": <number>,
-    "mean_k_filtering": <integer>
+    "mean_k_filtering": <integer>,
+    "max_dist_from_plane_mm": <integer>,
+    "ground_plane_normal_vec": <integer>,
+    "ground_angle_tolerance_degs": <>
```
Comment (npentrel) [inline docs/services/vision/segmentation.md:56]:
```suggestion
    "ground_angle_tolerance_degs": <number>
```

File: docs/services/vision/segmentation.md [hunk 4]
```diff
@@ -50,7 +50,10 @@ In your vision service's panel, fill in the **Attributes** field.
     "min_points_in_plane": <integer>,
     "min_points_in_segment": <integer>,
     "clustering_radius_mm": <number>,
-    "mean_k_filtering": <integer>
+    "mean_k_filtering": <integer>,
+    "max_dist_from_plane_mm": <integer>,
+    "ground_plane_normal_vec": <integer>,
```
Comment (npentrel) [inline docs/services/vision/segmentation.md:55]:
```suggestion
        "ground_plane_normal_vec": {
          "x": <integer>,
          "y": <integer>,
          "z": <integer>
        },
```

File: docs/appendix/release-notes.md [hunk 5]
```diff
@@ -252,7 +252,7 @@ You can replace existing TFLite classifiers by [configuring new ones in the UI](
 
 #### Radius Clustering 3D Segmenter configurations
 
-You can replace existing Radius Clustering 3D segmenters by [configuring new ones in the UI](/services/vision/segmentation/#configure-a-radius_clustering_segmenter) or you can update the [Raw JSON configuration of your robots](/manage/configuration/#the-config-tab):
+You can replace existing Radius Clustering 3D segmenters by [configuring new ones in the UI](/services/vision/segmentation/#configure-a-obstacles_pointcloud) or you can update the [Raw JSON configuration of your robots](/manage/configuration/#the-config-tab):
```
Comment (npentrel) [inline docs/appendix/release-notes.md:255]:
```suggestion
You can replace existing Radius Clustering 3D segmenters by [configuring new ones in the UI](/services/vision/segmentation/#configure-an-obstacles_pointcloud) or you can update the [Raw JSON configuration of your robots](/manage/configuration/#the-config-tab):
```

File: docs/services/vision/segmentation.md [hunk 6]
```diff
@@ -17,10 +17,10 @@ Any camera that can return 3D pointclouds can use 3D object segmentation.
 
 The types of segmenters supported are:
 
-- [**Radius clustering (`radius_clustering_segmenter`)**](#configure-a-radius_clustering_segmenter): A segmenter that identifies well separated objects above a flat plane.
+- [**Obstacles Pointcloud (`obstacles_pointcloud`)**](#configure-a-obstacles_pointcloud): A segmenter that identifies well separated objects above a flat plane.
```
Comment (npentrel) [inline docs/services/vision/segmentation.md:20]:
```suggestion
- [**Obstacles Pointcloud (`obstacles_pointcloud`)**](#configure-an-obstacles_pointcloud): A segmenter that identifies well separated objects above a flat plane.
```

**Pull request merged**